src/components/Card/Card.css

```diff
@@ -0,0 +1,55 @@
+/* Margin: 0.5em */
```

Comment:
The current colors are defined on https://bitnami-ui.now.sh/category/Foundations/Variables/#Colors. Could you update them to match the palette?